helpers/Users/updateUsers.js

```diff
@@ -0,0 +1,19 @@
+const { Users } = require("../../db/Models");
+
+async function UpdateUser(id, firstName, lastName) {
+  try {
+    const user = await Users.findOne({ where: { id: id } });
+    if(!user) {
+      return false;
+    }
+    user.firstName = firstName;
+    user.lastName = lastName;
+    await user.save();
+  } catch(err) {
+    console.error(`Unable to save user ${id}`);
```

Comment:
remove console.error

Reply:
https://github.com/The-Silent-Corner/sad-twt/pull/38/commits/c4f363c71f87c14edcda4ccab29f044dfc6d3963#diff-dfb8d86e4ac7fd43bd965ab73c07a694c1a2ff3a7e1155899b4d3f1e174a8e28L13